.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
+
+jobs:
+  release:
+    name: Create Release
+    if: github.event.base_ref == 'refs/heads/master'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get Version
+        id: get_version
+        run: echo ::set-output name=version::${GITHUB_REF/refs\/tags\//}
+
+      - name: Create & Publish Release
+        uses: release-drafter/release-drafter@v5.12.1
+        with:
+          config-name: release-drafter-config.yml
```

Comment:
Isn't the name of this file `release-drafter.yml` instead?

Reply:
Good catch. I started with a copy/paste of PBS-Java's tasks and forgot to change this back. Fixed.

Reply:
Don't we still need the `config-name` entry? Should we just change the file name?
```
20     with:
21 -     config-name: release-drafter-config.yml
   +     config-name: release-drafter.yml
22       name: ${{ steps.get_version.outputs.version }}
23       tag: ${{ steps.get_version.outputs.version }}
.github/workflows/release.yml 
```

Reply:
That is what I am wondering as well